packages/ember/tailwind-webpack/__snapshots__/index.test.ts.snap

```diff
@@ -0,0 +1,96 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`tailwind-webpack > applying to an ember app > works via CLI 1`] = `
+" package.json | 4 ++++
+ 1 file changed, 4 insertions(+)"
```

Comment:
only one file changed? maybe this is a signal that something is broken?

Reply:
Ok, so the issue was really that the files were not added. The trick needed is to add then via `git add . -N`, because they _must not_ be staged so that `ember-apply` recognizes them for future diffing.

I'm tempted to say that it would be better to actually use things in git stage, but that's another day's issue.

Happy to hear another round of feedback.